Dockerfile

```diff
@@ -0,0 +1,45 @@
+FROM golang:1.11-alpine3.7 as builder
+
+RUN apk add --no-cache --virtual .build-deps gcc musl-dev make git expat
+RUN apk add vips-dev --verbose --update-cache --repository https://dl-3.alpinelinux.org/alpine/edge/testing/ --repository https://dl-3.alpinelinux.org/alpine/edge/main
+
+ADD . /src
+WORKDIR /src
+RUN make dep
+RUN make test
+RUN make build
+
+#################
+FROM alpine:3.7
```

Comment:
The latest alpine version is 3.8. Is there a reason we pull 3.7?

Reply:
Not really it was a copy/paste from another dockerfile.